test/functional/feature_rbf.py

```diff
@@ -333,14 +330,9 @@ def test_new_unconfirmed_inputs(self):
         )["tx"]
         tx2_hex = tx2.serialize().hex()
 
-        # This will raise an exception
-        reject_reason = "replacement-adds-unconfirmed"
-        reject_details = f"{reject_reason}, replacement {tx2.txid_hex} adds unconfirmed input, idx 1"
-        res = self.nodes[0].testmempoolaccept(rawtxs=[tx2_hex])[0]
-        assert_equal(res["reject-reason"], reject_reason)
-        assert_equal(res["reject-details"], reject_details)
-        assert_raises_rpc_error(-26, f"{reject_details}", self.nodes[0].sendrawtransaction, tx2_hex, 0)
-
+        # This will not raise an exception
+        tx2_id = self.nodes[0].sendrawtransaction(tx2_hex, 0)
+        assert tx2_id in self.nodes[0].getrawmempool()
 
     def test_too_many_replacements(self):
         """Replacements that evict too many transactions are rejected"""
```

Comment:
In "Implement new RBF logic for cluster mempool" e6315c24326016cfaee5bd046e8b2e4e1088ac6b

nit: replace transactions with clusters here and other places
```suggestion
        """Replacements that evict too many clusters are rejected"""

```

Reply:
Fixed this in a couple of places, should be good now I think.